ccloud-observability/monitoring_configs/prometheus/prometheus.template.yml

```diff
@@ -41,16 +41,24 @@ scrape_configs:
         labels:
           env: 'dev'
 
-  - job_name: "ccloud-exporter"
+  - job_name: confluent-cloud
     static_configs:
       - targets:
-          - "ccloud-exporter:2112"
-        labels:
```

Comment:
label for env:dev is not included in the new `/export` version. does this affect the dashboards/display?

Reply:
Shouldn't have any impact, but it's a fair question - especially since we retain this label for the retained lag-exporter config.  Did you notice any impact?

I'll restore this label for now - but I think it's just an external label that would allow filtering in Prometheus.

Reply:
@JustBerkhout I've restored this label.